src/scss/abstracts/_mixins.scss

```diff
@@ -154,6 +152,11 @@
 		.card {
 			background-image: none;
 			box-shadow: 0 rem(4) rem(4) currentColor;
+
+			&:focus,
+			&:hover {
+				box-shadow: 0 0 0 rem(2) currentColor inset;
```

Comment:
This line should be `box-shadow: rem(10) rem(10) rem(10) currentColor;`

Reply:
The hover/focus style does look better by using the same css as the regular layout. Thanks, @TeddTech.

This pull request is ready for another review.